test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -1925,8 +1925,8 @@ protected NodeConfigurationSource getNodeConfigSource() {
             initialNodeSettings.put(NetworkModule.TRANSPORT_TYPE_KEY, getTestTransportType());
             initialTransportClientSettings.put(NetworkModule.TRANSPORT_TYPE_KEY, getTestTransportType());
         }
-        if (addTestZenDiscovery()) {
-            initialNodeSettings.put(TestZenDiscovery.USE_ZEN2.getKey(), true);
+        if (addTestZenDiscovery() == false) {
+            initialNodeSettings.put(TestZenDiscovery.USE_ZEN2.getKey(), false);
```

Comment:
this condition is non-sensical now. Only TestZenDiscovery understands the `TestZenDiscovery.USE_ZEN2` setting

Reply:
Yes, fixed in ace9194.